scripts/linters/js_ts_linter.py

```diff
@@ -1166,7 +1209,11 @@ def _lint_js_and_ts_files(self):
         if num_files_with_errors:
             for error in result_list:
                 python_utils.PRINT(error)
-                summary_messages.append(error)
+            # Extracting eslint output from index 0 because all error messages
+            # are present at the 0th index.
+            eslint_output = result_list[0]
```

Comment:
I'm not sure I follow. Isn't the for loop above called once per filepath? So result_list.append() would be called once per filepath, right? So you should have a for loop here instead?

Reply:
Used loop